### PR TITLE
N49: Ars Militaris

### PR DESCRIPTION
**Pull request by arsmilitaris**
Hey! Thanks for mentioning Ars Militaris. I've added a section on it. Hope everything is alright.

Part of #1440

### CODE REVIEW

File: content/news/049/index.md [hunk 1]
```diff
@@ -67,6 +67,38 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### Ars Militaris
+
+![Ars Militaris Logo](https://arsmilitaris.com/arsmilitaris_logo.png)
+
+Ars Militaris ([GitHub](https://github.com/arsmilitaris), [Discord](https://discord.gg/cdNDQsstgq), [Twitter](https://twitter.com/ArsMilitarisDev)) by @arsmilitaris
+is a Turn-based Tactics game set in the Ancient Rome era,
+being developed with Bevy Engine.
```
Comment (AngelOnFira) [inline content/news/049/index.md:76]: We put links at the bottom for cleanliness if you could switch that out :)
```
[This blog post][ars-militaris-blog-post]

...

[ars-militaris-blog-post]: https://example.com
```

**Review by AngelOnFira — CHANGES_REQUESTED** [review]
We might need to shorten this into only a couple paragraphs but otherwise LGTM!

**Comment by ozkriff** [comment]
:construction: editing

**Review by ozkriff — APPROVED** [review]
:+1: (i've embedded the image and made a bunch of style/fmt tweaks for consistency with the rest of the newsletter)

**Comment by ozkriff** [comment]
Thanks for the PR (and sorry for the delay)!

**Pull request merged**